java/src/jmri/jmrit/logixng/LogixNG_UserPreferences.java

```diff
@@ -1,15 +1,31 @@
 package jmri.jmrit.logixng;
 
+import java.util.HashMap;
+
 /**
  * This class is used for user preferences, like checkboxes for SystemNameAuto.
  * UserPreferencesManager uses reflection to access this class.
  *
  * @author Daniel Bergqvist   Copyright (C) 2021
  */
 public class LogixNG_UserPreferences {
-    
-    // No content in this class.
-    
-    // This class must have a default constructor.
```

Comment:
It would be good to keep this comment. If someone for whatever reason changes the constructor to take one or more parameters, this class would stop working.

Several constructors that takes parameters are fine, but there must be one constructor that doesn't take any parameters.

Reply:
It's the comment `This class must have a default constructor` that I refer to.

Reply:
@bobjacobsen 
If you want to merge this PR early, this comment is not important enough to keep this PR from being merged. So feel free to merge it if you want to.